src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,204 @@
+
+<CodeExample code={ComboboxMediaCode}>
+  <ComboboxMedia />
+</CodeExample>
+
+### Selection Logic
+
+The combobox can can be customized to include selection logic that highlights the first relevant
+result as the user types, allowing the user to hit enter to fully select the item. If this logic
+is not included, the user must manually select a result from the list.
+
+import ComboboxSelection from '../../examples/dropdowns/combobox/Selection.tsx';
+import ComboboxSelectionCode from '!!raw-loader!../../examples/dropdowns/combobox/Selection.tsx';
+
+<CodeExample code={ComboboxSelectionCode}>
+  <ComboboxSelection />
+</CodeExample>
+
+### Size
+
+Combobox can be displayed in either default or compact size.
+
+import ComboboxSize from '../../examples/dropdowns/combobox/Size.tsx';
+import ComboboxSizeCode from '!!raw-loader!../../examples/dropdowns/combobox/Size.tsx';
+
+<CodeExample code={ComboboxSizeCode}>
+  <ComboboxSize />
+</CodeExample>
+
+## How to use it well
+
+<BestPractice>
+  <Do imageSource={props.data.comboboxHighlightDo.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use balanced and appropriate typographic hierarchy.</p>
+  </Do>
+  <Dont imageSource={props.data.comboboxSearchableDont.childFile.childImageSharp.gatsbyImageData}>
+    <p>Don’t mix up typographic hierarchy when customizing item content.</p>
+  </Dont>
+</BestPractice>
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+The Combobox component follows this structure:
+
+```jsx
+<Dropdown>
+  <Field>
+    <Label />
+    <Hint />
+    <Combobox />
+  </Field>
+  <Menu>
+    <Item />
+    <Item />
+    {/* etc. */}
+  </Menu>
+</Dropdown>
+```
+
+### Combobox
+
+<Component components={props.data.mdx.components} componentName="combobox" />
+
+<PropSheet components={props.data.mdx.components} componentName="combobox" />
+
+### Dropdown
+
+<Component components={props.data.mdx.components} componentName="dropdown" />
+
+import Dropdowns from '../../fragments/dropdowns.mdx';
+
+<Dropdowns />
+
+<PropSheet components={props.data.mdx.components} componentName="dropdown" />
+
+### Field
+
+<Component components={props.data.mdx.components} componentName="field" />
+
+A Field provides accessibility attributes to the child [Combobox](#combobox) field by associating it
+with the corresponding [Label](#label) and [Hint](#hint).
+
+<PropSheet components={props.data.mdx.components} componentName="field" />
+
+### Hint
+
+<Component components={props.data.mdx.components} componentName="hint" />
+
+Nest a Hint within a [Field](#field) component.
+
+<PropSheet components={props.data.mdx.components} componentName="hint" />
+
+### Item
+
+<Component components={props.data.mdx.components} componentName="item" />
+
+Nest an Item within a [Menu](#menu) component. It requires a `value` prop, which is provided to
+the `onSelect` callback.
```

Comment:
Passive voice to active voice?

Reply:
updated to:
```
Nest an Item within a [Menu](#menu) component. It requires a `value` prop, which it provides to
the `onSelect` callback.
```

Reply:
![Screen Shot 2021-12-13 at 10 02 33 PM](https://user-images.githubusercontent.com/12474067/145925882-2e36f275-d189-4b26-9656-179b07dc3dc8.png)

Super odd, not updated on the PR, yet locally when pulled down, it reflects accurately to what was updated..